resources/js/components/discrete-bar.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
 
 export default function DiscreteBar(props: Props) {
   const current = Math.max(0, props.current);
-  const total = Math.max(1, clamp(props.total, current, props.total));
+  const total = Math.max(1, clamp(props.total, Math.min(current, props.total), props.total));
```

Comment:
I have a feeling it was just me at one point thought the bars should be at least as many as current count. Otherwise might as well just
```suggestion
  const total = Math.max(1, props.total);
```

Reply:
not sure how well hundreds of bars in a small space would render well 🤔